src/axcrypt_fmt_plug.c

```diff
@@ -58,9 +61,9 @@ john_register_one(&fmt_axcrypt);
 
 static struct fmt_tests axcrypt_tests[] = {
 	/*
-		formats can be:
-		$axcrypt$*version*iterations*salt*wrappedkey
-		$axcrypt$*version*iterations*salt*wrappedkey*key-file
+	 * Formats can be,
```

Comment:
@kholia I think this change was wrong, you should have kept the `:` instead of changing it into a `,`.

Reply:
I believe that both `:` and `,` are OK. I don't remember why I made this change though.